packages/github/package.json

```diff
@@ -38,12 +38,12 @@
     "url": "https://github.com/actions/toolkit/issues"
   },
   "dependencies": {
-    "@actions/http-client": "^2.0.1",
-    "@octokit/core": "^3.6.0",
-    "@octokit/plugin-paginate-rest": "^2.17.0",
-    "@octokit/plugin-rest-endpoint-methods": "^5.13.0"
+    "@actions/http-client": "^2.2.0",
+    "@octokit/core": "^4.2.4",
```

Comment:
Can we update to v5 and drop support for Node 16?

https://github.com/octokit/core.js/releases/tag/v5.0.0

https://github.com/actions/toolkit/issues/1542

Reply:
I u[dated to latest octokit